realm/realm-jni/src/io_realm_internal_tableview.cpp

```diff
@@ -131,6 +131,18 @@ JNIEXPORT void JNICALL Java_io_realm_internal_TableView_nativeClose(
     delete TV(nativeViewPtr);
 }
 
+JNIEXPORT jlong JNICALL Java_io_realm_internal_TableView_nativeGetCopy(
+    JNIEnv* env, jobject, jlong nativeViewPtr)
+{
+    if (!VIEW_VALID_AND_IN_SYNC(env, nativeViewPtr))
+        return 0;
+    try {
+        TableView* copiedTableView = new TableView(*TV(nativeViewPtr));
```

Comment:
If it's not clear from the documentation in the headerfile of core, make sure to make a PR/Issue in core to properly document the behaviour.